README.md

```diff
@@ -190,11 +190,16 @@ Our default branch is for development of our Boosted 5 release. Head to the [`v4
 ## Thanks
 
 <a href="https://www.browserstack.com/">
-  <img src="https://live.browserstack.com/images/opensource/browserstack-logo.svg" alt="BrowserStack Logo" width="192" height="42">
+  <img src="https://live.browserstack.com/images/opensource/browserstack-logo.svg" alt="BrowserStack" width="192" height="42">
 </a>
 
 Thanks to [BrowserStack](https://www.browserstack.com/) for providing the infrastructure that allows us to test in real browsers!
 
+<a href="https://www.netlify.com/">
```

Comment:
I think there's a dupe here, maybe miw them all together to fit to Bootstrap best ?

Reply:
Oh yes! Thanks! Fixed in a7152722c5ec8ee0e7180cfc650aea78df2b6176